### ISSUE TITLE
Add way to show/edit/add fax number

### ISSUE THREAD

**Issue by rugk** [mapsme/omim#4476]
I don't know whether the fax number is already shown (the OSM tag is `fax`), but I at least know that you cannot add it with the editor.

A small button near the phone number input (which adds a line below for fax) would be nice, so we can also add fax numbers.

**Comment by burivuh** [comment]
I can't guarantee we will add fax numbers (due to the mwm size issues) but we will check this case.

**Comment by thany** [comment]
What size issues? The maps aren't that big. And they'll keep growing anyway.

**Comment by burivuh** [comment]
Yes, and they are not that big (actually one the smallest in their class) because of 2 things:

1) Very strong accent on data compression
2) Very careful selection of what do our users really need on their maps

Fax requests aren't frequent

**Comment by thany** [comment]
1. Naturally. But phone storage gets bigger and bigger each year...
2. Issues like this are good evidence. Just try not to think you know what users want without asking first. Because fair is fair, you've made some rash decisions in the past.

**Comment by andree182** [comment]
#3161
